testing/go/functions_test.go

```diff
@@ -798,3 +798,204 @@ func TestSystemInformationFunctions(t *testing.T) {
 		},
 	})
 }
+
+func TestAggregateFunctions(t *testing.T) {
+	RunScripts(t, []ScriptTest{
+		{
+			Skip: true, // TODO: retrieve all row values into a function
+			Name: "string_agg",
+			SetUpScript: []string{
+				"CREATE TABLE test (id int, name text)",
+				"INSERT INTO test VALUES (1,'desk'), (2,'chair')",
+			},
+			Assertions: []ScriptTestAssertion{
+				{
+					Query: `select string_agg(name, ';;') from test;`,
+					Expected: []sql.Row{
+						{"desk;;chair"},
+					},
+				},
+				{
+					Query:    `select string_agg(null, null) from test;`,
+					Expected: []sql.Row{{nil}},
+				},
+				{
+					Query: `select string_agg(name, null) from test;`,
+					Expected: []sql.Row{
+						{"deskchair"},
+					},
+				},
+			},
+		},
+	})
+}
+
+func TestSchemaVisibilityInquiryFunctions(t *testing.T) {
+	RunScripts(t, []ScriptTest{
+		{
+			Skip:        true, // TODO: not supported
+			Name:        "pg_function_is_visible",
+			SetUpScript: []string{},
+			Assertions:  []ScriptTestAssertion{},
+		},
```

Comment:
We should add some assertions here. It's fine to use the default built-in functions for this.